### DISCUSSION TITLE
[Multicolor Highlights] as this plugin is abandoned, and creator approved forking it. Can anyone add ignore dead NPCs

### DISCUSSION THREAD

**Discussion by gytosas** [runelite/runelite#15937] [Q&A] [answered]
Other highlight plugins got this option.
Also this type of plugin for objects would help to.

**Comment by Enriath** [comment, marked as answer]
Issues and suggestions for hub plugins must be made on that plugin's repository; they are not maintained by us. The repo is usually accessible from the Support link on the plugin.
